language/en/phpbb_ideas_acp.php

```diff
@@ -22,6 +22,8 @@
 	// ACP module
 	'ACP_IDEAS_FORUM_ID'			=> 'Ideas forum',
 	'ACP_IDEAS_FORUM_ID_EXPLAIN'	=> 'Select the forum that Ideas topics will be posted to.',
+	'ACP_IDEAS_FORUM_SETUP'			=> 'Apply ideas forum settings',
+	'ACP_IDEAS_FORUM_SETUP_EXPLAIN'	=> 'Registered usergroup will not be able to start new topics in Ideas forum. Additionally, auto-pruning will be disabled for it.',
```

Comment:
How 'bout:
`Sets up the Ideas forum. Users will not be able to start new topics. Additionally, auto-pruning will be disabled.`